.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,32 +1,28 @@
 ---
-name: Bug report
-about: Create a report to help us improve
-title: "[BUG]"
-labels: ''
-assignees: ''
-
+name: Bug Report 🐛
+about: Something isn't working as expected? Here is the right place to report.
+labels: "type: kind/bug"
 ---
 
-<!--
-**Have a question or install problem?** Please use [Our Community Site](https://community.gravitational.com).
--->
-**Describe the bug**
-<!--A clear and concise description of what the bug is.-->
+### Description
+
+**What happened**:
 
-**To Reproduce**
-<!--Steps to reproduce the behavior:-->
+**What you expected to happen**:
 
-**Expected behavior**
-<!--A clear and concise description of what you expected to happen.-->
+**How to reproduce it (as minimally and precisely as possible)**:
 
-**Logs**
-<!--If applicable, add logs to help explain your problem.-->
+### Environment
 
-**Environment (please complete the following information):**
+ - Gravity version [e.g. 5.5.4]:
```

Comment:
As long as you're touching the file, you may modernize this.  5.5.4 was a long time ago.

Definitely don't bother if it would force you to go through another CI cycle.

Reply:
It's all good - I think it is well worth it. Hopefully this will encourage some folks to upgrade from 5.5.X once they see we are on 7.0.11!